apps/web/app/routes/resources+/validate-account.ts

```diff
@@ -0,0 +1,28 @@
+import { getUserAuthRole } from "@/.server/service/authorized-users";
+import { checkConnection } from "@/.server/service/clients";
+import { getUserFromRequest } from "@/utils/auth-headers";
+import { badRequest } from "@/utils/http";
+import { type ActionFunctionArgs, json, redirect } from "@remix-run/node";
+import { createUserRoleCookie } from "@server/utils/user-role-cookie";
+import { $path } from "remix-routes";
+
+export async function action({ request }: ActionFunctionArgs) {
+  const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
+  await sleep(5000); //FIXME remove
```

Comment:
This can be removed :)

Reply:
We should definitely add a CI that searches for FIXMEs 🙄